resources/clj-kondo.exports/lilactown/helix/clj_kondo/lilactown/helix.clj

```diff
@@ -99,7 +99,13 @@
                (api/token-node definer)
                component-name
                (filter some?
-                       [docstring metadata-map argvec
+                       [docstring metadata-map
+                        (if (and opts-node (get-in (api/sexpr opts-node) [:helix/features :define-factory]))
+                          (api/vector-node
+                            (concat (:children argvec)
+                                    [(api/token-node '&)
+                                     (api/token-node '_children)]))
```

Comment:
what does this fix, specifically?

Reply:
When you call the component with children `(component {} ($d "span" "foo"))`, kondo will no longer complain that the arity is wrong.